src/client/components/AchievementComponent/AchieveComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './AchieveComponent.styles.css';
+import keyImage from '../../assets/images/icons/kye-icon.svg';
+import xImage from '../../assets/images/icons/cross-icon.svg';
+import timerImage from '../../assets/images/icons/timer-icon.svg';
+import cupImage from '../../assets/images/icons/kup-icon.svg';
+
+export default function Achievement({ achieveTask, Ranking, timeSpent }) {
+  const minutes = Math.floor(timeSpent / 60);
+
+  return (
+    <div className="achieveMain">
+      <div className="AchieveBox">
+        <div className="AchieveCircle">
+          <img src={keyImage} alt="keyImage" />
+        </div>
+        <div>
+          <p className="wireframes">{achieveTask}</p>
+        </div>
+      </div>
+
+      <div className="AchieveBox">
+        <div className="AchieveCircle">
+          <img src={xImage} alt="xImage" />
+        </div>
+        <div>
+          <p className="wireframes">{5 - achieveTask}</p>
```

Comment:
What if i want to control the value 5 thru knobs? There is no way to control that. Maybe no 5 is maximum and fixed to 5 but if it is not i would like a knob for that. Other then that you did a good job 👍

Reply:
:)